### PR TITLE
check apply_planning_scene before creating collision-object-publisher/*ci*

### PR DESCRIPTION
**Pull request by k-okada**
this should avoid https://github.com/jsk-ros-pkg/jsk_robot/issues/1264 problem

### CODE REVIEW

**Review by knorth55 — APPROVED** [review]

File: pr2eus_moveit/euslisp/collision-object-publisher.l [hunk 1]
```diff
@@ -7,20 +7,28 @@
 
 (defmethod collision-object-publisher
   (:init (&key (service-name "apply_planning_scene")
-               (scene-service "get_planning_scene"))
+               (scene-service "get_planning_scene")
+	       (service-wait-time -1))
```
Comment (knorth55) [inline pr2eus_moveit/euslisp/collision-object-publisher.l:11]: please remove tab
```suggestion
               (service-wait-time -1))
```

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: pr2eus_moveit/euslisp/collision-object-publisher.l [hunk 2]
```diff
@@ -190,6 +201,9 @@
                          msg)
                      (send scene :world :collision_objects)))
        (send req :scene :is_diff t)
+       (unless servicename
+	 (ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
+	 (return-from :wipe-all-object t))
```
Comment (knorth55) [inline pr2eus_moveit/euslisp/collision-object-publisher.l:206]: please remove tabs.

```suggestion
         (ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
         (return-from :wipe-all-object t))
```

File: pr2eus_moveit/euslisp/collision-object-publisher.l [hunk 3]
```diff
@@ -209,6 +223,9 @@
                      (send scene :robot_state :attached_collision_objects)))
        (send req :scene :is_diff t)
        (send req :scene :robot_state :is_diff t)
+       (unless servicename
+	 (ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
+	 (return-from :wipe-all-attached-object t))
```
Comment (knorth55) [inline pr2eus_moveit/euslisp/collision-object-publisher.l:228]: please remove tabs.

```suggestion
         (ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
         (return-from :wipe-all-attached-object t))
```

File: pr2eus_moveit/euslisp/collision-object-publisher.l [hunk 4]
```diff
@@ -227,6 +244,9 @@
       (send msg :operation moveit_msgs::CollisionObject::*REMOVE*)
       (send req :scene :world :collision_objects (list msg))
       (send req :scene :is_diff t)
+      (unless servicename
+	(ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
+	(return-from :delete-object-by-msg))
```
Comment (knorth55) [inline pr2eus_moveit/euslisp/collision-object-publisher.l:249]: please remove tabs.

```suggestion
        (ros::ros-warn ";; collision-object-publisher ~A service could not found, so skipping calling service" servicename)
        (return-from :delete-object-by-msg))
```

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: pr2eus_moveit/euslisp/collision-object-publisher.l (hunk 1 above)
Comment (k-okada) [reply pr2eus_moveit/euslisp/collision-object-publisher.l]: thank you, removed all tabs

**Pull request merged**